src/js/study-browser/eva-study-browser-widget-panel.js

```diff
@@ -96,10 +96,13 @@ EvaStudyBrowserWidgetPanel.prototype = {
 
         this.panel = Ext.create('Ext.panel.Panel', {
             header: {
-                baseCls: 'eva-header-1',
-                titlePosition: 1
+                baseCls: '',
+                titlePosition: 1,
+                style: 'padding: 5px 0px 0px 7px;'
             },
-            title:'<nav aria-label="You are here:" role="navigation"><ul class="breadcrumbs"><li><a href="?Home">EVA</a></li><li><span class="show-for-sr">Current: </span>Study Browser</li></ul></nav>',
+            title:'<nav aria-label="You are here:" role="navigation"><ul class="breadcrumbs"><li><a href="?Home">EVA</a></li><li><span class="show-for-sr">Current: </span>Study Browser</li></ul></nav><h2>Study Browser</h2>' +
+            '<p>Search the Study browser warehouse using any combination of the filtering options on the left hand-side</p>' +
```

Comment:
"Search for studies archived at EVA" would be clearer for the end user. Our main goal here is to make clear that the study browser serves to **search for studies** because some people tried to use it to search for rs IDs.